src/Actions/ManagesEvents.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace TestMonitor\ActiveCampaign\Actions;
+
+use TestMonitor\ActiveCampaign\Resources\Event;
+
+trait ManagesEvents
+{
+    use Action;
+
+    /**
+     * This value is unique to ActiveCampaign account and can be found named "Event Key"
+     * on Settings > Tracking > Event Tracking inside ActiveCampaign account.
+     *
+     * @var string
+     */
+    public $eventKey;
+
+    /**
+     * This value is unique to ActiveCampaign account and can be found named "actid"
+     * on Settings > Tracking > Event Tracking API.
+     *
+     * @var string
+     */
+    public $actid;
+
+    /**
+     * Creates a new event (name only).
+     *
+     * @param string $name
+     *
+     * @return Event
+     */
+    public function createEvent($name)
+    {
+        $data = [
+            'name' => $name,
+        ];
+
+        $event = $this->post('eventTrackingEvents', ['json' => ['eventTrackingEvent' => $data]]);
+
+        return new Event($event['eventTrackingEvent']);
+    }
+
+    /**
+     * List all events.
+     *
+     * @return Event[]
+     */
+    public function events()
+    {
+        return $this->transformCollection(
+            $this->get('eventTrackingEvents'),
+            Event::class,
+            'eventTrackingEvents'
+        );
+    }
+
+    /**
+     * List name of all events.
+     *
+     * @return Event[]
+     */
+    public function eventNames()
+    {
+        $events = $this->get('eventTrackingEvents');
+
+        return array_map(function ($data) {
+            return $data['name'];
+        },
```

Comment:
While the linters say it's ok, the line below seems te be floating. I'd remove the newline.

Reply:
Done